scripts/tests/test_quac_matrix_adj_build.py

```diff
@@ -0,0 +1,44 @@
+import subprocess
+import pathlib
+import os
+import numpy as np
+
+DIR_PATH = pathlib.Path(__file__).parent.parent.parent
+
+str_path = str(DIR_PATH)
+
+
+def test_quac_mat_adj_build(script_runner):
+    wm_path = str_path + "/data/simplePhantoms/fanning_2d_5bundles/wm_vf.nii.gz"
+    fods_path = str_path + "/data/simplePhantoms/fanning_2d_5bundles/fods.nii.gz"
+    output_file = "test_output_graph.npz"
```

Comment:
If the output file is always the same for every test, I think you could move this line outside the functions, with DIR_PATH.